content/en/docs/Usage/backup.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Automated Backup
+linkTitle: Automated Backup
+date: 2024-12-21
+description: >
+  Information on the in-built backup system
+---
+
+Navidrome version 0.54.x introduces a backup feature that allows you to safeguard your music server's data. This guide will walk you through configuring backups using both the config.toml file and environment variables, as well as where to locate the backups.
```

Comment:
Note that it just backs up the database (not music). And probably also note the file names, and count.